aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -3427,8 +3429,33 @@ static void linalg_lstsq_out_info(
   auto input_working_copy = copyBatchedColumnMajor(input);
 
   // now the actual call that computes the result in-place (apply_lstsq)
-  lstsq_stub(input.device().type(), input_working_copy, solution, rank, singular_values, infos, rcond, driver);
-
+  if(driver == "gelsd" || driver == "gelss") {
```

Comment:
This doesn't do any tridiagonal work, so we should just use it when `gelss` is chosen.

Reply:
In that case, I'll also update this line ~3523
`(driver_str == "gelss" || driver_str == "gelssd"),`
to
`driver_str == "gelss"`